src/components/combobox/combobox.tsx

```diff
@@ -386,6 +388,7 @@ export class Combobox
   disconnectedCallback(): void {
     this.mutationObserver?.disconnect();
     this.resizeObserver?.disconnect();
+    this.el.removeEventListener("click", this.comboboxFocusHandler);
```

Comment:
Is this necessary? I see `onClick={this.clickHandler}` on the div that contains everything inside of the `<host>` and it seems to set focus already...

https://github.com/Esri/calcite-components/blob/c86f7910b9f007419063a863642a595c19ea3a71/src/components/combobox/combobox.tsx#L661-L668

If it is necessary, can you use an onClick on the host element to call the `comboboxFocusHandler`? This way the JSX handles creating and cleaning up the event. ;)

Reply:
there are actually two ways in combobox that focus is set:

setFocus - which puts focus on the textInput AND sets activeChipIndex and activeItemIndex to -1

comboboxFocusHandler - which only sets focus on the textInput

the click event on the div requires setFocus specifically whereas a click event on the combobox-item can not change activeChipIndex or activeItemIndex